tests/search.rs

```diff
@@ -84,9 +84,7 @@ fn simple() {
     assert_that(cargo_process("search").arg("postgres")
                     .arg("--host").arg(registry().to_string()),
                 execs().with_status(0)
-                       .with_stderr("\
-[UPDATING] registry `[..]`")
```

Comment:
Hm, looks like these `[UPDATING]` lines should still be present in the output somewhere?